libkbfs/cr_chains.go

```diff
@@ -864,6 +858,7 @@ func (ccs *crChains) getPaths(ctx context.Context, blocks *folderBlockOps,
 		for _, op := range chain.ops {
 			op.setFinalPath(p)
 		}
+		paths = append(paths, p)
```

Comment:
Oh actually I just ripped apart this function and rewrote it, and an identical line ended up in a different place.  Good catch, I will move it back!